### DISCUSSION TITLE
Using Shopify's Polaris with this package

### DISCUSSION THREAD

**Discussion by gbrits** [osiset/laravel-shopify#1096] [Q&A]
Hi there,

Surprised to see there's not more information around the use of Shopify's Polaris in tandem with this package.

I understand you can utilise your own stack here, VueJS, Livewire etc. But given the fact that Shopify encourages merchants to use the Shopify Polaris frontend framework has anyone used it in their stack & if so, how did you go about setting it up?

Thanks!

**Comment by Kyon147** [comment]
Hey @gbrits 

You can use polaris if you scaffold React with Laravel, as they only support React where Laravel prefers Vue. 

I do manage my own VueJS/Laravel Polaris package which you are free to use https://github.com/Kyon147/polaris-vue (Do bear in mind it might not be totally up to date with Shopify as I am just one man!)
